commands/shutdown.md

```diff
@@ -41,11 +46,28 @@ instead. This happens when:
 
```

Comment:
See above:
> there are two conditions

Should now be "three conditions".

Reply:
"several"?

Reply:
"all of the conditions below"... Many possibilitites

Reply:
Looking at this again, it doesn't seem like conditions that conditions have to be fulfilled. It's rather two different *situations* where exit is prevented. Maybe it should instead be sth like

"However there are two situations where it is unsafe to do so, and unless the **FORCE** modifier is specified, the **SHUTDOWN** command will be refused with an error instead. This happens when:"

WDYT?

Reply:
See commit "conditions -> situations"